apps/ftu/js/navigation.js

```diff
@@ -7,6 +7,7 @@
 'use strict';
 
 const DOGFOODSETTING = 'debug.performance_data.dogfooding';
+const METRICSSETTING = 'metrics.selectedMetrics.level';
```

Comment:
Nit: Having 'metrics' twice seems redundant. Perhaps there is an existing 'metrics' setting category. If not, would 'metrics.level' suffice?